resources/views/partials/main-content.blade.php

```diff
@@ -0,0 +1,66 @@
+<!-- Main Content -->
```

Comment:
Lets create a new file at '/resources/views/landing.blade.php`. Move this content to that file.
At the top add:
```suggestion
@extends('layout.welcome')
@section('content')
```
and at the bottom stop the section:
```
@stop
```